appIos/feed/feed/FeedScreen.swift

```diff
@@ -68,6 +74,7 @@ public struct FeedScreenContent: View {
 private struct FeedState: View {
     @EnvironmentObject var theme: Theme
     @Binding var selectedPostId: String?
+    @Binding var searchQuery: String
```

Comment:
At this point I'm unsure if we need the `searchQuery` in the `FeedState` - but if we use this state to track UI changes for re-renders, it's part of the lifecycle too.

Reply:
We will most likely show the current search query here but that will come from the view-model's state rather than the `@Binding` so I think it's okay to remove this for now and add later on in a separate PR (which will connect properly to the view-model).

Reply:
Done